tools/appliance/definitions/systemvmtemplate/configure_systemvm_services.sh

```diff
@@ -19,7 +19,7 @@
 set -e
 set -x
 
-CLOUDSTACK_RELEASE=4.6.0
+CLOUDSTACK_RELEASE=4.9.0
```

Comment:
Browsed merged PRs but nothing is merged that requires this. Or do I miss something?